.github/workflows/rust.yml

```diff
@@ -82,28 +75,25 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
 
   test-rs:
-    runs-on: buildjet-8vcpu-ubuntu-2204
+    runs-on: depot-ubuntu-24.04-8
     steps:
       - uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
       - uses: dtolnay/rust-toolchain@stable
-      - name: rust cache
-        uses: Swatinem/rust-cache@v2
-        with:
-          prefix-key: 'v3'
-          shared-key: 'rust'
-          cache-provider: 'buildjet'
-          save-if: ${{ !startsWith(github.ref, 'refs/heads/gh-readonly-queue') }}
-          workspaces: |
-            ./rust/main
-            ./rust/sealevel
       - name: Free disk space
         run: |
           sudo rm -rf /usr/share/dotnet
           sudo rm -rf /opt/ghc
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+      - name: Install mold linker
+        uses: rui314/setup-mold@v1
+        with:
+          mold-version: 2.0.0
+          make-default: true
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.8
```

Comment:
I think we used to use sccache prior to the new caching setup and it was pretty slow, concerned about increased misses. I assume we're switching to this because depot doesn't support `Swatinem/rust-cache@v2`

Reply:
if you look at the workflow summary this action provides the breakdown of hits/misses

Reply:
also moving to sccache gives us wayyyy more cache to play in GBs on depot